### ISSUE TITLE
User state file location

### ISSUE THREAD

**Issue by mikeymike** [php-school/php-workshop#155]
Currently user state is being saved in `.phpschool-save.json` inside the user's home dir while the workshop manager keeps itself self contained in the .php-school directory. 

I think it's potentially confusing having both the `.php-school` directory and then the `/phpschool-save.json` file both inside the home dir, maybe we should persist user state inside the `.php-school` directory instead ? 

Aside: There are potentially side effects here, we'd need to ensure the directory exists, we'd also need to add the current path to the legacy paths and process that as an array opposed to a single file as it is now. Potentially not worth effort.

**Comment by AydinHassan** [comment]
Yeah I'm not sure about this. That was a lot of crappy code to migrate the settings file last time.
